Cartography/View.swift

```diff
@@ -14,8 +14,6 @@ import Foundation
 
     extension UIView: LayoutItem {
         public func asProxy(context: Context) -> ViewProxy {
-            self.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
In 767821e4ccda5641d5edb2eca310c33af9dce7d2...da5e483451565ec1122c3f6ef0a2087468e1581e, why did `translatesAutoresizingMaskIntoConstraints = false` move to other methods?
It seems scattered than before 😕

Reply:
Yeah, sorry about that.

Here's the problem tho: this introduces a bug/breaking change on the library depending on how people use it, and I kinda discovered that by pointing out my fork to a project we are developing at our company.

When I centralized the `translatesAutoresizingMaskIntoConstraints` in the proxy generation step I was thinking hey this is going to make the library a lot more simple, but the thing is a lot of people use the syntax as in: 
```swift
constrain(view, superview) { view, superview in ... }
```
instead of:
```swift
constrain(view) { view in 
    view.top == view.superview!.top
}
```
in order to avoid IUOs

When we moved to the proxy code, the first case generated a side effect of setting up `translatesAutoresizingMaskIntoConstraints` in the superview, and this broke the layout of several root views in `UINavigationController` or any container view controller. 

I couldn't predict this side effect being generated by the code I made, so I changed back into the original behavior of the library, even if it looks ugly, to avoid more breaking changes.

Reply:
Thanks for explanation!
So, `translatesAutoresizingMaskIntoConstraints = false` impl is actually back to the original code now 🙄